src/_gettsim/transfers/wohngeld.py

```diff
@@ -41,21 +49,21 @@ def wohngeld_m_hh(
 
 
 def wohngeld_abzüge_st_sozialv_m(
-    eink_st_y_tu: float,
+    eink_st_y_sn: float,
     ges_rentenv_beitr_m: float,
     ges_krankenv_beitr_m: float,
     kind: bool,
     wohngeld_params: dict,
 ) -> float:
     """Calculate housing benefit subtractions on the individual level.
 
-    Note that eink_st_y_tu is used as an approximation for taxes on income (as mentioned
+    Note that eink_st_y_sn is used as an approximation for taxes on income (as mentioned
```

Comment:
This is actually a bad approximation as it may go systematically wrong for 1-earner couples (or secondary earner just doing a minijob) when first earner has sufficient income. I am not perfectly sure how this is being done in practice, though. Mayb open an issue with detailed Wohngeld questions?

Reply:
Sorry, I can't quite follow here. Is this related to the definition of "Steuern vom Einkommen" (WoGG) or the grouping on `sn` level (because same `sn_id` does not imply same `hh_id`)?

Reply:
Yes, I thought the "Steuern vom Einkommen" was defined at the individual level. If there is only one earner with sufficient income, both will pay income tax. But I'd think that subtraction will happen only once.

Reply:
Created #713